### PR TITLE
update for Phoenix 1.6

### PR DESCRIPTION
**Pull request by mkschell**
With an interest in crypto trading and learning Phoenix, this was a great opportunity for me to try upgrading an application to 1.6. I basically fumbled my way through this, so if I've done anything that is a bad practice for an Elixir or Phoenix PR, please let me know. I did get it working though ;-)

### CODE REVIEW

File: lib/ebisu_web/live/ticker_live.ex [hunk 1]
```diff
@@ -9,6 +9,7 @@ defmodule EbisuWeb.TickerLive do
   alias Ebisu.Bittrex.Ticker, as: BittrexTicker
   alias Ebisu.Cex
   alias Ebisu.Cex.Ticker, as: CexTicker
+  alias Poison
```
Comment (elpikel) [inline lib/ebisu_web/live/ticker_live.ex]: This is not used.

File: config/dev.exs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-use Mix.Config
+import Config
```
Comment (elpikel) [inline config/dev.exs:1]: Please also update `config/test.exs`.

**Review by elpikel — CHANGES_REQUESTED** [review]
Great work! There are two things that can be improved.

**Comment by mkschell** [comment]
Thanks for catching those. There are still a few tests that aren't passing. I am not proficient in Elixir/Phoenix testing methodology just yet, but am willing to dive in and figure it out if you would prefer for the PR to include all tests passing.

**Comment by elpikel** [comment]
Failing tests are calling external api, which has changed - so it is good that they are failing.

**Pull request merged**